src/Repository/CustomerRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of monofony.
+ *
+ * (c) Mobizel
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Repository;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+
+class CustomerRepository extends EntityRepository
+{
+    public function countCustomers(): int
+    {
+        return (int) $this->createQueryBuilder('o')
+            ->select('COUNT(o.id)')
+            ->getQuery()
+            ->getSingleScalarResult()
+            ;
```

Comment:
Semicolon here and in the method below is written differently :) We could improve consistency here. Can you run easy coding standard?

Reply:
I try to run coding standard, but no rules are defined about it.